### PR TITLE
WIP: Committees

### PR DESCRIPTION
**Pull request by cortner**
CC @casv2 @MatthiasSachs

### CODE REVIEW

**Comment by cortner** [comment]
So my plan now is to provide the basic committee evaluation that you requested + a backpropagation rule that allows you to compute check gradients from outside. You just need to pass in the co-tangent. We will discuss tomorrow and agree on the precise interface.

**Comment by cortner** [comment]
@casv2 @MatthiasSachs - `co_evaluate` is finished. Please take a look and see what you think.

**Comment by cortner** [comment]
This is ready to test. Also I'd be very grateful is somebody could read through the unit tests and mentally check for themseolves that these tests are sufficient to ensure correctness of the implementation. 

Only thing left to do is to write and read the committee. Everything else is bonus and can be done later.

**Comment by casv2** [comment]
Thanks very much, looks good!

File: test/test_committee.jl [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+##
+
+@info("Basic test of PIPotential construction and evaluation")
+maxdeg = 10
+r0 = rnn(:Cu)
+rcut = 2.5 * r0 
+basis = ACE1.Utils.rpi_basis(; species=:Cu, N=3, r0=r0, maxdeg=maxdeg, rcut=rcut)
+len = length(basis) 
+NCO = 12 
+co_c = Diagonal(1 ./ (1:len).^2) * randn(len, 12)
```
Comment (MatthiasSachs) [inline test/test_committee.jl:24]: `randn(len, NCO)` instead of `randn(len, 12)` ?

**Comment by MatthiasSachs** [comment]
Looks good to me! I left one minor comment.

**Comment by cortner** [comment]
FIO is done, please review and let me know if/when you are happy for me to merge this and tag as v0.10.0

**Pull request merged**